hosting_providers/index.md

```diff
@@ -4,25 +4,14 @@ title: Does My Hosting Provider Offer HTTPS?
 class: hosting
 ---
 
-## How did you set up your website?
-
-If you have a website, you maybe went through a process that included:
-<ol>
-  <li>
-    Buying a {% include tooltip.html term-name="domain-name" text="domain" %} from a company (yourwebsite.com)
-  </li>
-  <li>
-    Choosing a {% include tooltip.html term-name="server" text="server" %} to host that website. This often involves picking a {% include tooltip.html term-name="hosting-provider" text="hosting provider" %}.
-    {% include hosting_provider_graphic.html %}
-  </li>
-  <li>
-    Often, people will pick a {% include tooltip.html term-name="shared-hosting" text="shared hosting product" %} from the hosting provider, which may include an HTTPS certificate as part of their package. This page is for these users.
-  </li>
-</ol>
-
-## Do you use shared hosting?
-
-If you’re not using the shared hosting product, or if your provider doesn’t appear here, check out our [instructions](/instructions) for using Certbot or [get help](/help).
+<div class="shared-hosting">
+  <h3>This page is for users of {% include tooltip.html term-name="shared-hosting" text="shared hosting" %}.</h3>
+  <br>
+
+  If you're not using the shared hosting product, or if your provider doesn't appear here, check out our <a href="/instructions">instructions</a> for using Certbot or <a href="/help">get help</a>. Note that your {% include tooltip.html term-name="shared-hosting" text="shared hosting provider" %} is not the same as your {% include tooltip.html term-name="domain-registrar" text="domain registrar" %}.
```

Comment:
What do you think about nixing the second glossary link for shared-hosting here?

Reply:
I think it'd be useful to include the sentence "Note that your shared hosting provider
is not the same as your domain registrar.", without re-highlighting "shared hosting". Lots of newbies conflate the two.

Reply:
I just mean not making the second use of the term link to the glossary again.

Reply:
Oh, okay!